src/main/java/com/matera/cadastrocentral/address/AddressDTO.java

```diff
@@ -0,0 +1,93 @@
+package com.matera.cadastrocentral.address;
+
+import com.matera.cadastrocentral.client.Client;
+
+import java.util.UUID;
+
```

Comment:
Maybe you can add here the `@Data` and `@NoArgsConstructor` and delete the getters and setters